src/main/java/de/komoot/photon/query/PhotonRequestFactory.java

```diff
@@ -17,12 +18,22 @@
 public class PhotonRequestFactory {
     private final LanguageChecker languageChecker;
     private final static GeometryFactory geometryFactory = new GeometryFactory(new PrecisionModel(), 4326);
+    
+    protected static HashSet<String> m_hsRequestQueryParams = new HashSet<>(Arrays.asList("lang", "q", "lon", "lat", "limit", "distanceSort", "osm_tag"));
```

Comment:
If you care - at the end this is a matter of taste. We could also use distancesort.